service/src/main/kotlin/xyz/lebkuchenfm/Application.kt

```diff
@@ -56,20 +70,63 @@ fun Application.module() {
     val commandExecutorService = CommandExecutorService(textCommandParser, commandProcessorRegistry, commandPrompt)
 
     val discordClient = DiscordClient(environment.config, commandExecutorService)
-
     launch { discordClient.start() }
 
     install(ContentNegotiation) {
         json()
     }
 
+    install(Sessions) {
+        val days30 = 30 * 24 * 60 * 60 * 1000L
```

Comment:
thats in millisecond but variable below is `maxAgeInSeconds`

Reply:
You're absolutely right!